### ISSUE TITLE
ipvs proxier didn't delete ipvs rule on host when svc is deleted before pod

### ISSUE THREAD

**Issue by ogre0403** [kubernetes/kubernetes#89141]
**What happened**:

We use kube-proxy in IPVS mode, and upgrade from v1.12.5 to v1.13.10 last month. 

In v1.12.5 ~ 1.12.7, we note that ipvs rule will be removed if svc is deleted  whenever svc is deleted before or after pod deleting.

However, after upgrading to v.13.10, ipvs rule is NOT removed when svc is deleted FIRST then delete pod. And we also notice that v1.12.8 has the same behavior. 

**What you expected to happen**:
When svc is deleted, ipvs rule should be removed.

**How to reproduce it (as minimally and precisely as possible)**:
```shell
$ kubectl run nginx --image nginx --port 80
$ kubectl expose deployment nginx --port 51113 --target-port 80 --external-ip 172.22.2.141
$ kubectl delete svc nginx
$ kubectl delete deployment nginx 
```
After svc is deleted, ipvs rule is still exist. 
```
$ ipvsadm -L -n |grep -A1 172.22.2.141
TCP  172.22.2.141:51113 rr
  -> 10.244.124.218:80            Masq    1      0          0
```

![ipvs](https://user-images.githubusercontent.com/9261760/76732072-c3bfb700-6799-11ea-9b9d-1736f88fe41b.gif)

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
 1.12.8 & 1.13.10
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
  CentOS 7
- Kernel (e.g. `uname -a`):
  3.10.0-862.el7.x86_64
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:

**Comment by SataQiu** [comment]
Thanks for reporting this @ogre0403 !
v1.13 has exceeded the maintenance scope, maybe you can try a newer version.
I have tested v1.17.3, the problem can not be reproduced, everything looks fine.
I'll test v1.13.10 ASAP.

**Comment by SataQiu** [comment]
/cc @aojea
I can't reproduce the problem.
```sh
[root@test-01 ~]# kubectl version
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.2", GitCommit:"66049e3b21efe110454d67df4fa62b08ea79a19b", GitTreeState:"clean", BuildDate:"2019-05-16T16:23:09Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.10", GitCommit:"37d169313237cb4ceb2cc4bef300f2ae3053c1a2", GitTreeState:"clean", BuildDate:"2019-08-19T10:44:49Z", GoVersion:"go1.11.13", Compiler:"gc", Platform:"linux/amd64"}
[root@test-01 ~]# kubectl run nginx --image nginx --port 80
kubectl run --generator=deployment/apps.v1 is DEPRECATED and will be removed in a future version. Use kubectl run --generator=run-pod/v1 or kubectl create instead.
deployment.apps/nginx created
[root@test-01 ~]# kubectl expose deployment nginx --port 51113 --target-port 80 --external-ip 192.168.137.139
service/nginx exposed
[root@test-01 ~]# ipvsadm -L -n |grep -A1 192.168.137.139
TCP  192.168.137.139:51113 rr
TCP  10.96.0.1:443 rr
  -> 192.168.137.139:6443         Masq    1      0          23        
TCP  10.96.0.10:53 rr
[root@test-01 ~]# kubectl delete svc nginx
service "nginx" deleted
[root@test-01 ~]# ipvsadm -L -n |grep -A1 192.168.137.139
  -> 192.168.137.139:6443         Masq    1      0          27  
```

**Comment by ogre0403** [comment]
@SataQiu thanks for reminding me try newer version.   

This should be a bug and is already fixed in PR #80942
